src/jshint.js

```diff
@@ -4378,6 +4378,10 @@ var JSHINT = (function() {
     } else {
       // ImportClause :: NamedImports
       advance("{");
+      // Breaking change
```

Comment:
So this is the same for imports, but currently it is allowed so unlike exports its a breaking change?

It would be good to have this in and then a undoing commit - we can then revert the undoing for v3

@jugglinmike do you agree empty imports and exports should be warned?